content/en/blog/_posts/2021-09-10-kubernetes-in-kubernetes-and-pxe-bootable-servers-farm.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: blog
+title: "Kubernetes-in-Kubernetes and PXE bootable servers farm"
+date: 2021-09-10T22:47:55+02:00
+---
+
+**Author**: Andrei Kvapil (WEDOS)
+
+![Kubernetes in Kubernetes](/images/blog/2021-09-10-kubernetes-in-kubernetes-and-pxe-bootable-servers-farm/godeeper.png)
+
+
+When you have two own data centers, thousands of physical servers, virtual machines and hosting for hundreds of thousands sites, Kubernetes can actually simplify the management of all these things. As practice has shown, by using Kubernetes, you can declaratively describe and manage not only applications, but also the infrastructure itself. I work for the largest Czech hosting provider **WEDOS Internet a.s** and today I'll show you two of my projects — [Kubernetes-in-Kubernetes](https://github.com/kvaps/kubernetes-in-kubernetes) and [Kubefarm](https://github.com/kvaps/kubefarm).
```

Comment:
```suggestion
When you own two data centers, thousands of physical servers, virtual machines and hosting for hundreds of thousands sites, Kubernetes can actually simplify the management of all these things. As practice has shown, by using Kubernetes, you can declaratively describe and manage not only applications, but also the infrastructure itself. I work for the largest Czech hosting provider **WEDOS Internet a.s** and today I'll show you two of my projects — [Kubernetes-in-Kubernetes](https://github.com/kvaps/kubernetes-in-kubernetes) and [Kubefarm](https://github.com/kvaps/kubefarm).
```
A minor change that you might want to add. Just a grammar enhancement in the sentence :)